pkgs/by-name/dn/dns-over-https/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "dns-over-https";
+  version = "2.3.7";
+
+  src = fetchFromGitHub {
+    owner = "m13253";
+    repo = pname;
+    tag = "v${version}";
+    hash = "sha256-3mg4kDlIqv+psQU/FxA9TksGVAYaOymEDpAhMBZuqKI=";
+  };
+
+  vendorHash = "sha256-Dun1HWjV44PTpnijWBpGXqmKxtUw+Qu8rqsyMhEpkb4=";
+
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
+  subPackages = [
+    "doh-client"
+    "doh-server"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
```

Comment:
Missing `meta.mainProgram`.

Reply:
It contains two programs. I don't know how to set it.